docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -226,8 +226,9 @@ To restore using Restic, set these variables (in a Unix-based system; for Window
 
 [source,bash]
 ....
+kubectl port-forward svc/minio 9000:9000 &
```

Comment:
```suggestion
kubectl port-forward svc/minio 9000:9000 &
MINIO_PORT=$!
```
Then in the cleanup somewhere:
```
kill $MINIO_PORT
```

Reply:
which port is now relevant? 9000 or 9001? why the introduction to 9001 if not used?

Reply:
9001 is used for the Web-GUI
9000 is for the API. To restore we need the API port, but there is a use case to check in localhost:9001 in the GUI.

Suggested changes done with https://github.com/k8up-io/k8up/pull/666/commits/e12a60b13853dc56abc4e35635709d6e3a5c624d